### PR TITLE
New Article: Automate Downloading Logs from the Live Environment

### LINKED ISSUE #527: Automate Downloading Logs from the Live Environment - use rsync to download logs
Revise script located in [Automate Downloading Logs from the Live Environment](https://pantheon.io/docs/articles/sites/downloading-live-error-logs/) so that rsync is used. 

Update existing doc with new script.

### PR DESCRIPTION
**Pull request by rachelwhitton**
This PR contributes to #527 and adds a new doc: **Automate Downloading Logs from the Live Environment**

@ari-gold can you review/provide feedback? Thank you!

### CODE REVIEW

File: source/docs/articles/sites/downloading-live-error-logs.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Automate Downloading Logs from the Live Environment
+description: Learn how to automate downloading logs from the Live environment of your site for debugging.
+keywords: debug, debugging sites, debug sites, debugging site, debugging mysql, debug sql, troubleshoot mysql, troubleshoot sql, database logs, db logs, where are db logs stored, where are database logs, live logs, download logs, download log, access logs, get logs
+---
+
+Error logs help you find, debug, and isolate current or potential problems on your live site. You can automate the process of accessing and maintaining these logs with a simple script.
```
Comment (ari-gold) [inline source/docs/articles/sites/downloading-live-error-logs.md]: Logs aren't just error logs (for example nginx-access.log)

**Comment by rachelwhitton** [comment]
Thanks @ari-gold! I removed "error" from the opening statement. Does everything else look okay?

File: source/docs/articles/sites/downloading-live-error-logs.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Automate Downloading Logs from the Live Environment
+description: Learn how to automate downloading logs from the Live environment of your site for debugging.
+keywords: debug, debugging sites, debug sites, debugging site, debugging mysql, debug sql, troubleshoot mysql, troubleshoot sql, database logs, db logs, where are db logs stored, where are database logs, live logs, download logs, download log, access logs, get logs
+---
+
+Logs help you find, debug, and isolate current or potential problems on your live site. You can automate the process of accessing and maintaining these logs with a simple script.
+
+## Prerequisites
+The script requires passwordless access to your codebase, and you can do this by adding an SSH key to your Pantheon user account. See [Generating SSH Keys](/docs/articles/users/generating-ssh-keys/) for more information.
```
Comment (ari-gold) [inline source/docs/articles/sites/downloading-live-error-logs.md]: This isn't exactly a prerequisite. It's optional. If you don't have a SSH key added, you'll be prompted interactively for your Pantheon password.

File: source/docs/articles/sites/downloading-live-error-logs.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+```
+## Collect Logs
+Download logs by executing the script from within the `site-logs` directory:
+```
+sh collect-logs.sh
+```
+You can now access the logs from within the `site-log` directory. More than one directory is generated for sites that use multiple application containers.
+
+## See Also
+- [Debugging Sites with Log Files](/docs/articles/sites/debugging-sites-with-log-files/)
```
Comment (ari-gold) [inline source/docs/articles/sites/downloading-live-error-logs.md]: I think the Debugging Site with Log file article should link here and not duplicate same content. Probably good to include that change in this same PR?

Maybe we can come back to trying with Rsync, which is I think what the request was in #527?

**Pull request merged**